Gemfile.lock

```diff
@@ -9,62 +9,57 @@ GEM
       tzinfo (~> 1.1)
     celluloid (0.16.0)
       timers (~> 4.0.0)
-    chunky_png (1.3.3)
-    coffee-script (2.3.0)
+    chunky_png (1.3.4)
+    coffee-script (2.4.1)
       coffee-script-source
       execjs
-    coffee-script-source (1.8.0)
-    compass (1.0.1)
+    coffee-script-source (1.9.1.1)
+    compass (1.0.3)
       chunky_png (~> 1.2)
-      compass-core (~> 1.0.1)
+      compass-core (~> 1.0.2)
       compass-import-once (~> 1.0.5)
       rb-fsevent (>= 0.9.3)
       rb-inotify (>= 0.9)
       sass (>= 3.3.13, < 3.5)
-    compass-core (1.0.1)
+    compass-core (1.0.3)
       multi_json (~> 1.0)
       sass (>= 3.3.0, < 3.5)
     compass-import-once (1.0.5)
       sass (>= 3.2, < 3.5)
-    em-websocket (0.5.1)
-      eventmachine (>= 0.12.9)
-      http_parser.rb (~> 0.6.0)
     erubis (2.7.0)
-    eventmachine (1.0.4)
-    execjs (2.2.2)
-    ffi (1.9.6)
+    execjs (2.5.2)
+    ffi (1.9.8)
     haml (4.0.6)
       tilt
     hike (1.2.3)
     hitimes (1.2.2)
     hooks (0.4.0)
       uber (~> 0.0.4)
-    http_parser.rb (0.6.0)
```

Comment:
:smile: thanks for keeping a look out! I think it's fine though, these are the unused gems removed in #197